packages/manager/modules/pci/src/projects/project/training/job.class.js

```diff
@@ -58,6 +60,21 @@ export default class Job {
     }
   }
 
+  isPreRunning() {
+    return (
+      this.status.state === STATE_ENUM.QUEUED ||
+      this.status.state === STATE_ENUM.PENDING ||
+      this.status.state === STATE_ENUM.INITIALIZING
+    );
```

Comment:
```suggestion
    return [STATE_ENUM.QUEUED, STATE_ENUM.PENDING, STATE_ENUM.INITIALIZING].includes(this.status.state);
```

Reply:
Fixed along with other similar functions in the file